lib/react_on_rails/version_checker.rb

```diff
@@ -20,21 +24,37 @@ def initialize(node_package_version)
     # unless the user really knows what they're doing. So we will give a
     # warning if they do not.
     def raise_if_gem_and_node_package_versions_differ
```

Comment:
is anybody code still calling this one?

we'll need a reminder for the next big update? Maybe leave a PR pending?

Reply:
We can create a follow-up PR and leave it pending.